CHANGELOG

```diff
@@ -122,15 +122,14 @@ v 7.9.4
   - Fixed issue where only 25 commits would load in file listings
   - Fix LDAP identities  after config update
 
-v 7.9.3
-  - Contains no changes
+v 7.9.3 (GitLab-EE)
```

Comment:
This changelog should only contain Community edition changes so this is not needed